Say/ViewController.swift

```diff
@@ -9,12 +9,16 @@
 import Cocoa
 // import SayKit
 
-
 /// Main window of the application
 class MainWindow: NSWindow {
     @IBOutlet var speechToolbarItem: NSToolbarItem! = nil
+    @IBOutlet var pauseToolbarItem: NSToolbarItem! = nil
     @IBOutlet var exportToolbarItem: NSToolbarItem! = nil
     @IBOutlet var openToolbarItem: NSToolbarItem! = nil
+    @IBOutlet var stopToolbarItem: NSToolbarItem! = nil
+    //@IBOutlet var statusItem: NSStatusItem! = nil
+    //@IBOutlet var statusMenu: NSMenu! = nil
```

Comment:
Do we need to remain these comments? It looks unnecessary to remain these two lines ;(